docs/output.md

```diff
@@ -143,10 +143,11 @@ Optionally, ASV sequences can be filtered for rRNA sequences identified by Barrn
 <summary>Output files</summary>
 
 - `barrnap/`
-  - `ASV_seqs.ssu.fasta`: Fasta file with filtered ASV sequences.
-  - `AASV_table.ssu.tsv`: Counts for each filtered ASV sequence.
   - `rrna.<kingdom>.gff`: GFF3 output for rRNA matches per kingdom, where kingdom is one of `bac,arc,mito,euk`.
-  - `stats.ssu.tsv`: Tracking read numbers through filtering, for each sample.
+  - `summary.tsv`: Summary of evalues for each ASV and kingdom
+  - `ASV_seqs.ssu.fasta`: Fasta file with filtered ASV sequences, only if `--filter_ssu` is set.
+  - `AASV_table.ssu.tsv`: Counts for each filtered ASV sequence, only if `--filter_ssu` is set.
```

Comment:
Is the double A in AASV_table.ssu.tsv intentional?

Reply:
Thats a typo, thanks, will change!